src/main/java/OperatorTest.java

```diff
@@ -0,0 +1,39 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import util.Operator;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class OperatorTest {
+    private Operator operator;
+    @BeforeEach
+    void setUp() {
+        operator = new Operator();
+    }
+    @DisplayName("+")
+    @ParameterizedTest
+    @CsvSource("'1','2'")
```

Comment:
각 double을 Char 형식으로 묶을 필요 없습니다.
```suggestion
    @CsvSource("1,2")
```